labellines/test.py

```diff
@@ -164,7 +164,7 @@ def test_dateaxis_advanced(setup_mpl):
     ax.xaxis.set_major_locator(DayLocator())
     ax.xaxis.set_major_formatter(DateFormatter("%Y-%m-%d"))
 
-    labelLines(ax.get_lines())
+    labelLines(ax.get_lines(), xvals=(dates[0], dates[-1]))
```

Comment:
Shouldn't this place the first label on 2018/11/01 and the second on 2018/11/03?

Reply:
That's true for a list input (`xvals=[dates[0], dates[-1]]`), but the tuple input adds padding automatically